CoffeeChat/Controller/SettingsViewController.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  SettingsViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 10/7/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+import GoogleSignIn
+
+protocol PausePearDelegate: class {
+    func presentPausePear()
+    func pausePearAction(data: String)
+    func removePausePear()
+}
+
+class SettingsViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let settingsTableView = UITableView()
+    private var pauseBlurEffect: UIBlurEffect!
+    private var pauseVisualEffectView: UIVisualEffectView!
+    private var pausePearView: PausePearView!
+    
+    // MARK: - Private Data Vars
+    private var settingOptions: [SettingOption] = [
+        SettingOption(image: "pausePear", text: "Pause Pear", hasSwitch: true, switchOn: false),
+        SettingOption(image: "editAvailability", text: "Edit Availabilities", hasSwitch: false, switchOn: false),
+        SettingOption(image: "socialMedia", text: "Connect Social Media", hasSwitch: false, switchOn: false),
+        SettingOption(image: "aboutPear", text: "About Pear", hasSwitch: false, switchOn: false),
+        SettingOption(image: "logout", text: "Log Out", hasSwitch: false, switchOn: false)
+    ]
+    let settingsReuseIdentifier = "settingsReuseIdentifier"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "Settings"
+        view.backgroundColor = .backgroundLightGreen
+        
+        settingsTableView.backgroundColor = .backgroundLightGreen
+        settingsTableView.separatorStyle = .none
+        settingsTableView.showsVerticalScrollIndicator = false
+        settingsTableView.allowsSelection = true
+        settingsTableView.delegate = self
+        settingsTableView.dataSource = self
+        settingsTableView.register(SettingOptionTableViewCell.self, forCellReuseIdentifier: settingsReuseIdentifier)
+        view.addSubview(settingsTableView)
+        
+        setupNavigationBar()
+        setupPausePear()
+        setupConstraints()
+    }
+    
+    private func setupNavigationBar() {
```

Comment:
alot of this code is repeated as well. if we could find a way to condense it at all that would be ideal!